cloud/js-green-licenses.json

```diff
@@ -0,0 +1,30 @@
+{
+  "packageWhitelist": [
+    /* ourselves */
+    "arcs@file:../",
+    /* no license */
+    "base62",
+    /* License in LICENSE.md: W3C 3-clause BSD License */
+    "abab",
+    /* WTFPL */
+    "left-pad",
+    /* no license */
+    "prompts",
+    /* WTFPL */
+    "truncate-utf8-bytes",
+    /* WTFPL */
+    "utf8-byte-length",
+    /* WTFPL */
+    "argsarray",
+    /* WTFPL */
+    "expand-template",
+    /* WTFPL */
+    "spark-md5",
+    /* Apache 2.0, but not discoverable */
+    "@tensorflow/tfjs-node",
```

Comment:
We should consider removing this for now? #3330

Reply:
Done.